enterprise/frontend/src/metabase-enterprise/embedding_iframe_sdk/index.ts

```diff
@@ -1,6 +1,10 @@
 import { PLUGIN_EMBEDDING_IFRAME_SDK } from "metabase/plugins";
 import { hasPremiumFeature } from "metabase-enterprise/settings";
 
+import { DashboardSubscriptionsButton } from "../../embedding-sdk-ee/subscriptions/DashboardSubscriptionsButton";
+
 if (hasPremiumFeature("embedding_simple")) {
-  PLUGIN_EMBEDDING_IFRAME_SDK.hasValidLicense = () => true;
+  PLUGIN_EMBEDDING_IFRAME_SDK.isEnabled = () => true;
```

Comment:
https://metaboat.slack.com/archives/C063Q3F1HPF/p1764747739501169

Basically, rename this so it's consistent with all other plugins out there